x-pack/solutions/security/plugins/security_solution/public/common/components/user_privileges/user_privileges_context.tsx

```diff
@@ -70,22 +79,31 @@ export const UserPrivilegesProvider = ({
     [kibanaCapabilities]
   );
 
+  const rulesPrivileges = useMemo(() => {
+    return {
+      read: readRules,
+      edit: editRules,
+    };
+  }, [readRules, editRules]);
+
   const contextValue = useMemo(
     () => ({
       listPrivileges,
       detectionEnginePrivileges,
       endpointPrivileges,
-      kibanaSecuritySolutionsPrivileges,
+      siemPrivileges,
       timelinePrivileges,
       notesPrivileges,
+      rulesPrivileges,
```

Comment:
nit: We could get rid of the `Privileges` suffix. The hook name already implies we are fetching privileges. With that in mind the following line

```ts
const canEditRules = useUserPrivileges().rulesPrivileges.edit;
```

could be simplified to

```ts
const canEditRules = useUserPrivileges().rules.edit;
```

Reply:
We end up breaking this out in the next [RBAC PR for exceptions](https://github.com/elastic/kibana/pull/243095) and using the `rulesPrivileges` name as more of an umbrella term for rules permissions and its subfeatures.

e.g. 
```
const canEditRules = useUserPrivileges().rulesPrivileges.rules.edit
```
and
```
const canEditExceptions = useUserPrivileges().rulesPrivileges.exceptions.edit
```